i18n/nl/docusaurus-plugin-content-docs/current/staff/coaches/attendance.md

```diff
@@ -0,0 +1,31 @@
+---
+sidebar_position: 3
+---
+
+# Aanwezigheid
+
+Aanwezigheidsregistratie in Jarvis is een optionele functie die in 2 verschillende modi kan functioneren.
+
+Beide modi tonen de student een grafiek op hun dashboard van de geregistreerde aanwezigheid.
+
+![dashboard widget voor studentenaanwezigheid](/img/coaches/attendance/student-dashboard.png)
+
+Daarnaast, als de chatfunctie is ingeschakeld, ontvangt de student geautomatiseerd chatbericht als
```

Comment:
Misschien hiervoor ook standaard opzet gaan bedenken? Tabel met: 'related features'? 

Chat: de student ontvangt geautomatiseerd bericht via de chat als er een negatieve registratie plaats vind (ziek, te laat of ongeoorloofd afwezig)
Coach pilot: veel maatstaven in de coach pilot werken alleen goed als de studenten hun aanwezigheid geregistreerd wordt. Kijk bij de documentatie van de coach pilot voor meer informatie hierover.

Reply:
Ik denk dat je er niet aan ontkomt om te linken naar andere docs pagina's. Denk dat het hebben van veel cross links juist wel goed is.

Als het een visuele tabel boven aanwordt dat kan wel wat groot worden als je zoveel info er in kwijt wilt en het kan zijn dat het niet lekker leest. Maar is ook een beetje de insteek, optimaliseren we voor cover to cover (waar deze schrijf stijl dichterbij zit) of optimaliseren we voor scannen.

Ik had de indruk dat onze doelgroep, niet superhard wil scannen dus dat logisch lezen bij cover to cover belangrijker is.